tutor-contrib-harmony-plugin/tutor_k8s_harmony_plugin/patches/k8s-services

```diff
@@ -1,20 +1,26 @@
----
 # For multi-instance clusters. Allow one central load balancer on the cluster to
 # handle HTTPS certs and forward traffic to each Open edX instance's Caddy
 # instance.
-{%- set HOSTS = [LMS_HOST, CMS_HOST, PREVIEW_HOST, MFE_HOST] + K8S_HARMONY_INGRESS_HOST_LIST %}
+{%- set HOSTS = [LMS_HOST, CMS_HOST] + K8S_HARMONY_ADDITIONAL_INGRESS_HOST_LIST %}
+{%- set tmp = HOSTS.append(PREVIEW_HOST) if PREVIEW_HOST is defined %}
+{%- set tmp = HOSTS.append(MFE_HOST) if MFE_HOST is defined %}
```

Comment:
Maybe we could include MinIO in this list? I think is fair to include it as a default because it's kind of required if you are not using the object storage from the cloud provider.

Reply:
`MINIO_HOST` and `MINIO_CONSOLE_HOST` are added.